nsv13/code/modules/overmap/knpc.dm

```diff
@@ -65,7 +67,8 @@ GLOBAL_LIST_EMPTY(knpcs)
 /datum/component/knpc/Initialize()
 	if(!iscarbon(parent))
 		return COMPONENT_INCOMPATIBLE
-	failsafe_trust = rand((2 SECONDS), (5 SECONDS))
+	if(failsafe_trust_variance)
+		failsafe_trust = rand(failsafe_trust, (failsafe_trust + failsafe_trust_variance))
```

Comment:
Since failsafe trust variance is pretty much always set there's not really a reason to do an if check I think. Even 0 will still just resolve.

Reply:
I guess, yeah